Core/Inc/pdu.h

```diff
@@ -41,17 +43,18 @@ int8_t read_tsms_sense(pdu_t* pdu, bool* status);
 
 /* Functions to Read Status of Various Stages of Shutdown Loop */
 typedef enum {
-	CKPT_BRB_CLR,	/* Cockpit BRB */
-	SIDE_BRB_CLR,	/* Side BRB */
-	INTERTIA_SW_OK, /* Inertia Switch */
-	BOTS_OK,		/* Brake Over Travel Switch */
-	BPSD_OK,		/* Brake System Plausbility Device */
-	IMD_OK,			/* Insulation Monitoring Device */
-	BMS_OK,			/* Battery Management System (Shepherd) */
-	TSMS,			/* Tractive System Main Switch */
-	HVD_INTLK_OK,	/* HVD Interlock */
-	HVC_INTLK_OK,	/* HV C Interlock*/
-	MAX_SHUTDOWN_STAGES
+	CKPT_BRB_CLR = 0,	/* Cockpit BRB */
+    BMS_OK = 2,			/* Battery Management System (Shepherd) */
+    INTERTIA_SW_OK = 3, /* Inertia Switch */
+    SPARE_GPIO1_OK = 4,
+    IMD_OK = 5,			/* Insulation Monitoring Device */
+    BPSD_OK = 8,		/* Brake System Plausbility Device */
+	BOTS_OK = 13,		/* Brake Over Travel Switch */
+	HVD_INTLK_OK = 14,	/* HVD Interlock */
+	HVC_INTLK_OK = 15,	/* HV C Interlock*/
+    SIDE_BRB_CLR,	/* Side BRB */
+    TSMS,			/* Tractive System Main Switch */
+	MAX_SHUTDOWN_STAGES = 9
 } shutdown_stage_t;
```

Comment:
values are pins

Reply:
Since I made such changes to the fuse and shutdown enums, do we need to adjust how they are iterated in monitor.c lines 236 and 272 @nwdepatie